CHANGELOG.md

```diff
@@ -1,14 +1,15 @@
 ## Release History
-* 0.2.4 Change handlebars dependency as peerDependency to be able to install custom supported version
 
-* 0.2.3 Don't override options.jquery if jQuery is global (@xcambar - #9)
+- 0.3.0 Add extra config parameter to be able to define custom handlebars compiler
```

Comment:
I don't think this is a breaking change any longer now that the library can fall back to the built-in Handlebars if `options.handlebars.module` is not provided?